ui/src/app/tasks/task-execution/task-execution.component.ts

```diff
@@ -118,6 +118,19 @@ export class TaskExecutionComponent implements OnInit {
     return args.map((arg) => arg.split('='));
   }
 
+  /**
```

Comment:
It would be useful to move this generic function into a common component so that they can be used by some other components as well.